commit cfc617beaace4e99c293ed095dae9315b4869747
Author: Dave Eddy <dave@daveeddy.com>
Date:   2019-05-28T13:42:06-04:00 (4 months ago)
    
    MANTA-4303 electric-boray fails if the optional "fast" config is not present

diff --git a/lib/server.js b/lib/server.js
index d98496d..41792ba 100644
--- a/lib/server.js
+++ b/lib/server.js
@@ -221,6 +221,7 @@ LimitOffsetStream.prototype.getNextRecord =
 function createServer(options, callback) {
     assert.object(options, 'options');
     assert.optionalObject(options.fast, 'options.fast');
+    options.fast = options.fast || {};
     assert.optionalNumber(options.fast.client_crc_mode,
         'options.fast.client_crc_mode');
     assert.optionalNumber(options.fast.server_crc_mode,
